skype_send_message.py

```diff
@@ -5,23 +5,29 @@
 parent_parser = argparse.ArgumentParser(add_help=False)
 parent_parser.add_argument('--username', action="store")
 parent_parser.add_argument('--password', action="store")
-print("Welcome to 'Skype send message'. ")
-username = input("Type your username of Skype: ")
-password = keyring.get_password("Skype", username)
-if (password != keyring.get_password("Skype", username)):
-    password = input("Type your password of Skype: ")
+parent_parser.add_argument('--servicename', action="store")
+parent_parser.add_argument('--user_conversation', action="store")
+parent_parser.add_argument('--message_conversation', action="store")
+def main(username, user_conversation, message_conversation):
+    print("Welcome to 'Skype send message'. ")
+    servicename = ""
```

Comment:
@idumenskyi why not servicename(aka `platform`) is not passed to the `main` function?

Reply:
@ekondrashev  because, servicename used only keyring

Reply:
@idumenskyi that is not true, you're using `servicename` at 14th line